### PR TITLE
fix: check if blog post authors are specified in fetch-team-data script

### PR DESCRIPTION
**Pull request by snitin315**
Refers https://github.com/eslint/website/pull/934

### CODE REVIEW

File: tools/fetch-team-data.js [hunk 1]
```diff
@@ -145,8 +145,10 @@ async function fetchBlogAuthors() {
         const contents = await fs.readFile(filename, "utf8");
         const { data: frontmatter } = matter(contents);
 
-        for (const username of frontmatter.authors) {
-            authors[username] = await fetchUserProfile(username);
+        if (Array.isArray(frontmatter.authors)) {
+            for (const username of frontmatter.authors) {
+                authors[username] = await fetchUserProfile(username);
+            }
```
Comment (harish-sethuraman) [inline tools/fetch-team-data.js:151]: May be something like this? 
```suggestion
        const authors = frontmatter.authors;

        if (Array.isArray(authors)) {
            for (const username of authors) {
                authors[username] = await fetchUserProfile(username);
            }
```

Comment (snitin315) [reply tools/fetch-team-data.js:151]: Same stuff. It's just stylistic personal preferences.

Comment (harish-sethuraman) [reply tools/fetch-team-data.js:151]: Yep kinda, but when there might be a change in the value we might want to change at all occurrences. But we can avoid that by doing like this and change at one occurrence alone?

**Review by nzakas — APPROVED** [review]
LGTM.

**Pull request merged**